### ISSUE TITLE
cant purchase anything

### ISSUE THREAD

**Issue by en0rgy** [josephchang10/text-to-speech-support#5]
simply because the purchase plans are not listed! only can choose 3 days trial that already passed, nothing happens when i click it as its already over. please list the membership plans. smh

**Comment by normgoss** [comment]
I agree. The **Purchase Button** is missing, therefore, I cannot buy the subscription. Please ensure that the **Purchase Button** is available all the time during and after the 3-day trial period. Thank you!
